core/input/input.cpp

```diff
@@ -1207,6 +1190,38 @@ void Input::_axis_event(int p_device, JoyAxis p_axis, float p_value) {
 	parse_input_event(ievent);
 }
 
+void Input::_update_action_strength(Action &p_action, int p_event_index, float p_strength) {
+	ERR_FAIL_INDEX(p_event_index, MAX_EVENT + 1);
```

Comment:
```diff
diff --git a/core/input/input.cpp b/core/input/input.cpp
index d844d8b7c6..257452b3d8 100644
--- a/core/input/input.cpp
+++ b/core/input/input.cpp
@@ -1191,7 +1191,7 @@ void Input::_axis_event(int p_device, JoyAxis p_axis, float p_value) {
 }
 
 void Input::_update_action_strength(Action &p_action, int p_event_index, float p_strength) {
-	ERR_FAIL_INDEX(p_event_index, MAX_EVENT + 1);
+	ERR_FAIL_INDEX(p_event_index, (int)MAX_EVENT + 1);
 
 	float old_strength = p_action.strengths[p_event_index];
 	p_action.strengths[p_event_index] = p_strength;
@@ -1207,7 +1207,7 @@ void Input::_update_action_strength(Action &p_action, int p_event_index, float p
 }
 
 void Input::_update_action_raw_strength(Action &p_action, int p_event_index, float p_strength) {
-	ERR_FAIL_INDEX(p_event_index, MAX_EVENT + 1);
+	ERR_FAIL_INDEX(p_event_index, (int)MAX_EVENT + 1);
 
 	float old_strength = p_action.raw_strengths[p_event_index];
 	p_action.raw_strengths[p_event_index] = p_strength;
```
Or cast the other one, but then you need to use `ERR_FAIL_UNSIGNED_INDEX`.

Reply:
Actually I should just change the constant to `int`. I originally used unsigned it, because it did something different.